### PR TITLE
#167725236 send account verification email

### PR DESCRIPTION
**Pull request by wangonya**
Install requirements:

```
(env) $ pip install -r requirements.txt
```

Start redis server:

```
$ redis-server
```

Start celery worker:

```
(env) $ celery -A recipes worker -l info
```

Endpoint:

```
http://127.0.0.1:8000/api/auth/verify-account/?token=example_token
```

Screenshot 2019-08-25 at 20 42 51
<p align="center">
Screenshot 2019-08-25 at 20 43 15
</p>

### CODE REVIEW

File: recipes/authentication/migrations/0001_initial.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='user',
             name='role',
-            field=models.ForeignKey(default=0, on_delete=django.db.models.deletion.DO_NOTHING, to='authentication.Role'),
+            field=models.ForeignKey(default=2, on_delete=django.db.models.deletion.DO_NOTHING, to='authentication.Role'),
```
[the remark this answers is not in the document]
Comment (wangonya) [reply recipes/authentication/migrations/0001_initial.py]: This is inevitable

File: recipes/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+# This will make sure the app is always imported when Django starts
+from .celery import app as celery_app
```
[the remark this answers is not in the document]
Comment (wangonya) [reply recipes/__init__.py]: This is necessary

**Comment by willeswa** [comment]
Nice work on the template

File: recipes/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonathanmusila) [reply recipes/__init__.py]: How is it being used?

[the remark this answers is not in the document]
Comment (wangonya) [reply recipes/__init__.py]: @jonathanmusila look at the comment just above it

**Pull request merged**